LetsGame/js/scriptssp.js

```diff
@@ -0,0 +1,72 @@
+/* Schere Stein Papier*/
+let _userPoint = 0; //Benutzerpunkte
+let _pcPoint = 0; //Pc-Punkte
+let _roundsToPlay = 0; //Rundenzähler
+

```

Comment:
Zwinge den User dazu die Versuche einzulesen. 
Du lest es in der ersten Runde ein, und das ist Grundsätzlich in Ordnung - aber man übersieht das Feld.
Bzw - was ist wenn der User 0 eingibt? dann sollte eine schöne Fehlermeldung kommen, dass er mindestens 1 eingeben muss.